tests/ci/README.md

```diff
@@ -0,0 +1,135 @@
+
+### In a fresh virtual machine
+
+You can run the tests inside a virtual machine (VM), by running the following:
+
+```shell
+tmt run -a -vvv \
+    provision --how virtual \
+    --image http://cloud.centos.org/centos/8-stream/x86_64/images/CentOS-Stream-GenericCloud-8-20210603.0.x86_64.qcow2 \
+    --memory 8192 \
+    --disk 60
+```
+
+The `provision --how virtual` makes sure it runs in a VM. But the default
+option is a Fedora VM. To force it to use **CentOS Stream 8** you'll need
+to indicate a Centos Stream 8 image.
+
+To be able to run some of the steps, you'll need at least 8 GB of RAM and
+around 60 Gb of disk. The previous command takes care of all those limits.
+
+**NOTE**: For using the `virtual` provisioner you should install the `tmt-all`
+package.
+
+### In a CentOS 8 or CentOS Stream 8 machine
+
+It also can run in the same machine, but it needs to be CentOS 8 or CentOS
+Stream 8. It doesn't matter if the machine is virtual (like with Vagrant or
+libvirt), or if it's a bare-metal, but it needs to be able to run virtual
+machines inside.
+
+The steps to run the tests are:
+
+1. Connect to the machine you want to use.
+1. Make sure you have `tmt` and `git` installed (`dnf install -y tmt git`)
+1. Clone this repo: `git clone https://github.com/osbuild/automotive-ci.git`
+1. Run `tmt` locally with the following command:
+
+```shell
+tmt run -a -vvv provision --how local
+```
+
+### In a remote machine
+
+You'd run in a remote machine. You just need to have `tmt` installed locally
+and `ssh` access to the remote machine. And that remote machine needs to run
+on CentOS 8 or CentOS Stream 8.
+
+To run the scripts remotely you need to run the following:
+
+```shell
+tmt run -a -vvv provision --how connect --guest $REMOTE_SERVER --key ~/.ssh/id_rsa
+```
+
+That command will run the scripts on the `$REMOTE_SERVER` using the user `root`
+and the ssh key `~/.ssh/id_rsa`, but you can also use user and password:
+
+```shell
+tmt run -a -vvv provision --how connect --guest $REMOTE_SERVER --user myuser --password secret
+```
+
+### Debug when something fails
+
+All the previous examples have the flag `-v` (verbose) but with 3 `v`, which
+means more verbose. That is very useful to see what is happening.
+
+But sometimes, you'll need to debug what went wrong. `tmt` will call the
+`clean_up` script when something fails and you want to be able to explore.
+To be able to explore and debug you have the command `login` and you can
+use it as follows to debug in case something fails.
+
+```shell
+tmt run -a -vvv provision --how local login --step execute --when fail
+```
+
+### Passing ENV variables
+
+Sometimes you'll need to pass an ENV variable to the scripts, you can do it
+with the option `-e` or `--environment`. It can be used multiple times,
+but it's important to put it before the command `provision`:
+
+```shell
+tmt run -a -vvv -e AWS_ACCESS_KEY_ID=secret_key_id -e AWS_SECRET_ACCESS_KEY=secret_key provision --how local
+```
+
```

Comment:
maybe would be nice to add a link to the official docs is someone wants to further read?
https://tmt.readthedocs.io/en/latest/

Reply:
Yes, I forgot :facepalm: 
Thanks!

Reply:
Oh, I remembered. I put the link at the introduction, but it's not very explicit. I'll point to the official documentation for further read.